marlowe-runtime/web-server/Language/Marlowe/Runtime/Web/Server.hs

```diff
@@ -79,12 +81,14 @@ compile $ SelectorSpec "server"
   , "http" ≔ Inject ''ServeRequest
   , "api" ≔ Inject ''ApiSelector
   , ["contract", "indexer"] ≔ Inject ''ContractHeaderIndexerSelector
+  , ["history"] ≔ Inject ''HistoryClientSelector
```

Comment:
This is new syntax to me. Is this part of the `DataKinds` extension? Is it representing a type as a value?